pkgs/development/libraries/capstone/default.nix

```
@@ -22,6 +23,12 @@ stdenv.mkDerivation rec {
     fixDarwinDylibNames
   ];
 
+  cmakeFlags = [
+    "-DBUILD_SHARED_LIBS=${if static then "OFF" else "ON"}"
+    "-DCAPSTONE_BUILD_CSTOOL=ON"
+    "-DCAPSTONE_BUILD_TESTS=${if finalAttrs.finalPackage.doCheck then "ON" else "OFF"}"
+  ];
+
   doCheck = true;
 
```

```suggestion
prePatch = ''
    sed -i 's#VERSION 5.0#VERSION 5.0.1#' CMakeLists.txt
'';
```

This fixes the capstone.pc generated by cmake.